### PR TITLE
Create login partial view

### PR DESCRIPTION
**Pull request by suecarmol**
## Description
Create the upper part of the new Wikipedia Library Homepage, compatible with RTL languages and with different mobile devices.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
This is part of our site's redesign efforts.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T282115](https://phabricator.wikimedia.org/T282115)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
Tested in different screen sizes and different languages In the web browser (Firefox)

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

**Laptop screen**
Screen Shot 2021-05-24 at 19 38 06

**Mobile screen**
Captura de pantalla 2021-05-25 - 19 38 28

**Tablet screen**
Captura de pantalla 2021-05-25 - 19 38 34

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/static/css/new-local.css
Comment (jsnshrmn) [inline TWLight/static/css/new-local.css:40]: A small bottom margin for the login section will keep the bottom of the login button from disappearing in mobile view, and doesn't have much of an impact  on the others. Could go in a media query or be approached differently.
```suggestion
*/
.login-section{
    margin-bottom: 5px;
}
```

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
This is looking good on desktop and tablet, but there are a few minor issues for mobile.

**Review by jsnshrmn — APPROVED** [review]

**Comment by jsnshrmn** [comment]
This looks good to me!  Waiting on other team feedback to merge. Feel free to do so if you get the green light!

File: TWLight/static/css/new-local.css
Comment (suecarmol) [reply TWLight/static/css/new-local.css:12]: Thanks, that solved it!

File: TWLight/static/css/new-local.css
Comment (suecarmol) [reply TWLight/static/css/new-local.css:47]: Done!

**Comment by jsnshrmn** [comment]
This is looking good, though it looks like the rtl stylesheet needs to be regenerated.

**Comment by suecarmol** [comment]
> though it looks like the rtl stylesheet needs to be regenerated.

I don't understand this comment. I don't think I deleted the RTL stylesheet.

**Comment by jsnshrmn** [comment]
The RTL file looks out of sync with the main one, at least in the gh review diff. Sometimes gh gets a little wacky, so I can pull it down to double check

**Comment by suecarmol** [comment]
> The RTL file looks out of sync with the main one, at least in the gh review diff. Sometimes gh gets a little wacky, so I can pull it down to double check

Oh, got it! I found several redundant  css classes. Will push a new commit shortly

**Review by jsnshrmn — APPROVED** [review]
Looks great!

**Pull request merged**
